app/views/custom_styles/_primer_color_mapping.erb

```diff
@@ -5,15 +5,38 @@
       --body-background: var(--color-canvas-default);
       --body-font-color: var(--color-fg-default);
       --color-accent-fg: var(--content-link-color) !important;
-
-  <% DesignColor.overwritten.each do |design_color| %>
-    <% if design_color.variable == "alternative-color" %>
-        --color-btn-primary-bg:  <%= design_color.hexcode %> !important;
-        --color-btn-primary-hover-bg:  <%= design_color.darken 0.18 %> !important;
-    <% end %>
-    <% if design_color.variable == "content-link-color" %>
-        --color-accent-fg: <%= design_color.hexcode %> !important;
-    <% end %>
-  <% end %>
+  }
+  [data-light-theme=light_high_contrast]{
+    --button--alt-highlight-background-hover-color: var(--color-btn-primary-hover-bg);
+    --button--alt-highlight-background-color: var(--color-btn-primary-bg);
+    --button--alt-highlight-border-color: var(--main-menu-font-color);
+    --main-menu-hover-border-color: var(--color-fg-default) !important;
+    --avatar-border-color: var(--color-avatar-border);
+    --toolbar-item--border-color: var(--color-border-default);
+    --button--border-color: var(--color-border-default);
+    --table-border-color: var(--color-border-default);
+    --list-item-hover-color: var(--color-action-list-item-default-active-border);
+    --inplace-edit--border-color: var(--color-border-default);
+    --ck-color-toolbar-border: var(--color-border-default);
+    --ck-color-base-border: var(--color-border-default);
+    --button--active-border-color: var(--color-border-default);
+    --button--background-hover-color: var(--color-btn-hover-bg);
+    --button--background-color: var(--color-btn-bg);
+    --toolbar-item--bg-color: var(--color-btn-bg);
+    --header-bg-color: var(--AppHeader-bg);
+    --header-item-font-color: var(--color-fg-default);
+    --header-item-bg-hover-color: var(--color-action-list-item-default-hover-bg);
+    --header-item-font-hover-color: var(--color-fg-default);
+    --header-border-bottom-color: var(--color-border-muted) !important;
+    --main-menu-bg-color: var(--AppHeader-bg);
+    --main-menu-hover-font-color: var(--color-fg-default);
+    --main-menu-bg-selected-background: var(--color-action-list-item-default-hover-bg);
+    --main-menu-bg-hover-background: var(--color-action-list-item-default-hover-bg);
+    --main-menu-font-color: var(--color-fg-default);
+    --main-menu-selected-font-color: var(--color-fg-default);
+    --main-menu-border-color: var(--color-border-muted);
+    --main-menu-resizer-color: var(--color-border-muted);
+    --main-menu-bg-selected-border: var(--color-border-muted);
+    --header-border-bottom-width: 1px;
```

Comment:
Please take the time and sort these so the belonging variables are shown below each other (so that all button styles are in a block, all main menu styles, all header styles etc.). Currently it is really hard to gasp all variables when trying to understand the effects on one part of the application.